test/framework/toolchain.py

```diff
@@ -189,7 +169,7 @@ def test_validate_pass_by_value(self):
     def test_optimization_flags(self):
         """Test whether optimization flags are being set correctly."""
 
-        flag_vars = ['CFLAGS', 'CXXFLAGS', 'FFLAGS', 'F90FLAGS']
+        flag_vars = ['CFLAGS', 'CXXFLAGS', 'FCFLAGS', 'FFLAGS', 'F90FLAGS']
```

Comment:
Like you said, it doesn't really matter. In fact, the order here matches the order of the `COMPILER_FLAGS` list.